drush/sites/self.site.yml

```diff
@@ -11,4 +11,15 @@
 #  host: stage.domain.com
 #  user: www-admin
 #  root: /path/to/drupal
-#  uri: http://stage.example.com
\ No newline at end of file
+#  uri: http://stage.example.com
+#
+# For Silta environments, you can use this wildcard alias.
+# Replace [repo-name] with the name of your repository, and leave out the square brackets.
+# Usage: drush @branch-name status
```

Comment:
For usage, would be good to note the possible forward slashes of branch-name `/` need to be replaced with a dash `-`.